students/7/main.js

```diff
@@ -16,8 +16,20 @@ const getUTCDateTime = () => {
         res(date);
     })
 }
+
+
 const ex2 = async () => {
-    console.log("TODO...")
+    let characters = await countCharsInBody(1);
+    console.log(characters)
+}
+
+const countCharsInBody = async (userId) => {
+    let count = 0;
+
+    let result = await axios.get(`https://jsonplaceholder.typicode.com/posts/${userId}`);
```

Comment:
Did we want to put this in a `try/catch` block?

Reply:
yes! adding now